core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -24,7 +24,7 @@ <h4 style="clear: left; display: inline-block;">Summary</h4>
         <th></th>
         <th>RDD Blocks</th>
         <th><span data-toggle="tooltip"
-                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory</span>
+                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory used/total</span>
```

Comment:
Because of this change, easier to understand for users and observation.Web UI with clarity and friendly.